Source/CesiumRuntime/Private/Tests/CesiumFeatureIdSpecUtility.h

```diff
@@ -0,0 +1,182 @@
+#pragma once
+
+#include "CesiumGltf/ExtensionExtMeshFeatures.h"
+#include "CesiumGltf/Model.h"
+#include <glm/glm.hpp>
+#include <vector>
+
+/**
+ * @brief Adds the buffer to the given primitive, creating a buffer view and
+ * accessor in the process.
+ * @returns The index of the accessor.
+ */
+int32_t AddBufferToPrimitive(
+    CesiumGltf::Model& model,
+    CesiumGltf::MeshPrimitive& primitive,
+    const std::string& type,
+    const int32_t componentType,
+    const std::vector<std::byte>&& values) {
+  CesiumGltf::Buffer& buffer = model.buffers.emplace_back();
```

Comment:
I think it's worthwhile to move the implementations to a .cpp file. It probably won't really make much difference in the grand scheme of Unreal Engine code, but less code in headers means less code that needs to be compiled when editing source files that use those headers, and that's particularly a good thing for headers meant to be used in tests, because we'd like the iteration time on changing tests to be as quick as possible.